manim/mobject/geometry.py

```diff
@@ -435,7 +435,7 @@ def __init__(self, start, end, angle=TAU / 4, radius=None, **kwargs):
             arc_height = radius - math.sqrt(radius ** 2 - halfdist ** 2)
             angle = math.acos((radius - arc_height) / radius) * sign
 
-        Arc.__init__(self, radius=radius, angle=angle, **kwargs)
+        super().__init__(radius=radius, angle=angle, **kwargs)
```

Comment:
Is this relevant for this PR?

Reply:
Not really, just caught it while something was breaking.

Reply:
I guess we could leave it in.